apis/cc-de_CH/atrium-api-cc-de_CH-common/src/main/kotlin/ch/tutteli/atrium/api/cc/de_CH/mapAssertions.kt

```diff
@@ -81,6 +81,15 @@ inline fun <K, reified V : Any, T: Map<K, V?>> Assert<T>.enthaeltNullable(keyVal
 fun <K> Assert<Map<K, *>>.enthaeltKey(key: K)
     = addAssertion(AssertImpl.map.containsKey(this, key))
 
+/**
+ * Makes the assertion that [AssertionPlant.subject] contains the given [key].
```

Comment:
Makes the assertion that [AssertionPlant.subject] does not contain the given [key].